data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,142 @@
+  mutate(season = as.integer(season)) %>%
+  filter(season != "SEASON")
+devtools::use_data(bachelorette, overwrite = TRUE)
+
+
+# candy-power-rankings -------------------------------------------------------------
+candy_rankings <- read_csv("data-raw/candy-power-ranking/candy-data.csv") %>%
+  clean_names() %>%
+  mutate(
+    chocolate = as.logical(chocolate),
+    fruity = as.logical(fruity),
+    caramel = as.logical(caramel),
+    peanutyalmondy = as.logical(peanutyalmondy),
+    nougat = as.logical(nougat),
+    crispedricewafer = as.logical(crispedricewafer),
+    hard = as.logical(hard),
+    bar = as.logical(bar),
+    pluribus = as.logical(pluribus)
+  ) %>%
+  mutate_at(competitorname, funs(gsub("Õ", "'", .))) ###########################
+
+devtools::use_data(candy_rankings, overwrite = TRUE)
+
+
+# chess-transfers ----------------------------------------------------------------
+chess_transfers <- read_csv("data-raw/chess-transfers/transfers.csv") %>%
+  clean_names() %>%
+  mutate(
+    transfer_date = as.Date(transfer_date, "%m/%d/%y"),
+    id = as.character(id)
+  )
+devtools::use_data(chess_transfers, overwrite = TRUE)
+
+# congress-generic-ballot --------------------------------------------------------
+# generic_polllist
+generic_polllist <- 
+  read_csv("https://projects.fivethirtyeight.com/generic-ballot-data/generic_polllist.csv")
+write_csv(generic_polllist, path = "data-raw/congress-generic-ballot/generic_polllist.csv")
+
+generic_polllist <- 
+  read_csv("data-raw/congress-generic-ballot/generic_polllist.csv") %>%
+  clean_names() %>%
+  mutate(
+    modeldate = as.Date(modeldate, "%m/%d/%Y"),
+    startdate = as.Date(startdate, "%m/%d/%Y"),
+    enddate = as.Date(enddate, "%m/%d/%Y"),
+    createddate = as.Date(createddate, "%m/%d/%Y"),
+    timestamp = parse_date_time(timestamp, "HMS dmY"),
+    subgroup = as.factor(subgroup),
+    pollster = as.factor(pollster),
+    grade = as.factor(grade),
+    population = as.factor(population),
+    poll_id = as.character(poll_id),
+    question_id = as.character(question_id)
+  )
+devtools::use_data(generic_polllist, overwrite = TRUE)
+
+# generic_topline
+generic_topline <- 
+  read_csv("https://projects.fivethirtyeight.com/generic-ballot-data/generic_topline.csv")
+write_csv(generic_topline, path = "data-raw/congress-generic-ballot/generic_topline.csv")
+
+generic_topline <- 
+  read_csv("data-raw/congress-generic-ballot/generic_topline.csv") %>%
+  clean_names() %>%
+  mutate(
+    modeldate = as.Date(modeldate, "%m/%d/%Y"),
+    timestamp = parse_date_time(timestamp, "HMS dmY"),
+    subgroup = as.factor(subgroup)
+  )
+devtools::use_data(generic_topline, overwrite = TRUE)
+
+
+#inconvenient-sequel ----------------------------------------------------------------
+ratings <- read_csv("data-raw/inconvenient-sequel/ratings.csv") %>%
+  mutate(category = as.factor(category))
+devtools::use_data(ratings, overwrite = TRUE)
```

Comment:
It seems when columns/variables start with numbers, you still need the tick marks to access them. Could you 
* rename all `1_votes` to `votes_1`
* same for `1_pct`
* Update the help file accordingly?